### PR TITLE
Add/correct data for worker support for various APIs

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR sets real values for worker support across various PRs.  The results were collected using tests from the mdn-bcd-collector project.

### CODE REVIEW

File: api/CustomEvent.json [hunk 1]
```diff
@@ -225,13 +225,13 @@
           "description": "Available in workers",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "43"
```
Comment (foolip) [inline api/CustomEvent.json:228]: Confirmed by the date of https://bugs.chromium.org/p/chromium/issues/detail?id=370011#c7 + https://www.chromium.org/developers/calendar

**Comment by foolip** [comment]
@vinyldarkscratch can you describe the methodology here? Have you manually tested and looked at the "Window exposure" variants of tests, or have you used a script and collected results? Depending on what you've done, I might not need to review very carefully.

**Comment by vinyldarkscratch** [comment]
I went through each of the APIs' tests in the collector and looked at the "Worker exposure" support entry.  After narrowing down the version number, I then put it in BCD!

**Comment by foolip** [comment]
Thanks @vinyldarkscratch! That sounds pretty time-consuming, but it should give the correct results!

File: api/WindowOrWorkerGlobalScope.json [hunk 2]
```diff
@@ -858,7 +858,7 @@
               "version_added": "8"
             },
             "samsunginternet_android": {
-              "version_added": "2.0"
+              "version_added": "1.5"
```
Comment (foolip) [inline api/WindowOrWorkerGlobalScope.json:861]: This seems like a random change, but indeed Chrome 24 would mirror to Samsung 1.5.

**Review by foolip — APPROVED** [review]

**Pull request merged**